### ISSUE TITLE
Support for multi flavor libraries

### ISSUE THREAD

**Issue by remcomokveld** [novoda/bintray-release#73]
I have a library which comes in multiple product flavors. I see in the code that there is a TODO in de AndroidArtifacts class when it comes to selecting the right output aar. The following code at the end of my build.gradle now allows me to publish various flavors of the library. Maybe you can use the same concept for automatically creating publish tasks for all product flavors in a library

```
project.afterEvaluate {

    project.android.libraryVariants.each { variant ->
        if (variant.buildType.name == "debug") return;
        Artifacts artifacts = project.plugins.hasPlugin('com.android.library') ? new AndroidArtifacts() : new JavaArtifacts()

        project.publishing.publications.create(variant.name, MavenPublication) {
            groupId project.publish.groupId
            artifactId project.publish.artifactId+"-"+variant.productFlavors.collect { it.name }.join("-")
            version = project.publish.publishVersion

            artifacts.all(it.name, project).each {
                if (it instanceof CharSequence) {
                    delegate.artifact "${project.buildDir}/outputs/aar/${project.name}-${variant.baseName}.aar"
                } else {
                    delegate.artifact it
                }
            }
            from artifacts.from(project)
        }
    }
}
```

**Comment by xrigau** [comment]
Hey @remcomokveld 

Many thanks for taking the time to create this issue!
Would you be up for creating a PR with your solution? otherwise we can look into it at some point but it'd be great if you want to contribute back to the repo :smile:

**Comment by remcomokveld** [comment]
Just created a PR https://github.com/novoda/bintray-release/pull/74 :)

**Comment by xrigau** [comment]
nice action! :+1:

**Comment by xrigau** [comment]
Thanks for your contribution @remcomokveld !!

I've just released 0.3.5 which contains this fix https://bintray.com/novoda/maven/bintray-release/0.3.5/view

**Comment by xrigau** [comment]
Fixed in #74

**Comment by GautamMandsorwale** [comment]
Hi @remcomokveld 

In the above code snippet by you, how do you resolve `'Artifacts', 'AndroidArtifacts'` symbols in the build.gradle file?  I get this `'Cannot resolve symbol AndroidArtifacts'`. 

Also I checked commits on master branch of the bintray-release repo (https://github.com/novoda/bintray-release/commit/fc8abf77b79629d5ea2b17b546acbcff0fb73032) and it seems, your above suggested way is refactored with use of `AndroidAttachments` . Which way to go then?

This might be a silly question, but I am struggling to find a way around this. Please guide.
